app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -112,6 +112,10 @@ class MainViewModel @Inject constructor(
         it?.let(MailboxController::getMailbox)
     }.asLiveData(ioCoroutineContext)
 
+    val currentMailboxLive = _currentMailboxObjectId.flatMapLatest { objectId ->
+        objectId?.let(MailboxController::getMailboxAsync)?.mapNotNull { it.obj } ?: emptyFlow()
+    }.asLiveData(ioCoroutineContext)
+
```

Comment:
This is used only in ThreadViewModel.

It should be moved to ThreadViewModel, and get the Mailbox via `AccountUtils.currentUserId` & `AccountUtils.currentMailboxId` instead of `_currentMailboxObjectId`.